marlowe-chain-sync/docs/marlowe-chain-sync.1

```diff
@@ -22,8 +22,8 @@ chain using the Filtered Chain Sync Protocol.
 
```

Comment:
Historically, we haven't created `man` pages for Marlowe tools. Do we know whether these will work properly on Windows and Mac? @ladamesny might want to test on Mac and @dino-iog on Windows.